libtcmu.c

```diff
@@ -590,6 +607,10 @@ static void remove_device(struct tcmulib_context *ctx,
 	if (ret < 0)
 		tcmu_err("could not cleanup caw lock %s: %d\n", dev_name, errno);
 
+	ret = pthread_mutex_destroy(&dev->format_lock);
+	if (ret < 0)
+		tcmu_err("could not cleanup format lock %s: %d\n", dev_name, errno);
```

Comment:
I think there was an error in the code right above this.

pthread_mutex_destroy does not set errno does it? The ret value will be a Exyz value.